.github/workflows/ci.yml

```diff
@@ -8,12 +8,19 @@ jobs:
   test:
     strategy:
       matrix:
-        node: [10.x, 12.x, 14.x, 16.x]
-    runs-on: ubuntu-latest
+        node: [14.18.0, 14, 16.0.0, 16]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
Appveyor is windows testing; no need to duplicate it here. Is there a good reason for testing macOS separately?